### PR TITLE
Lesson 02 print grid exercise script

### PR DESCRIPTION
**Pull request by bubblecfd**
add lesson 02 print grid exercise script.

### CODE REVIEW

File: students/tao_ye/lesson02/fizz_buzz.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    if (i % 3 == 0 and i % 5 == 0):
+        print("FizzBuzz")
+        continue
+    if (i % 3 == 0):
+        print("Fizz")
+        continue
+    if (i % 5 == 0):
+        print("Buzz")
+        continue
+    print(i)
```
Comment (charlesrobison) [inline students/tao_ye/lesson02/fizz_buzz.py:11]: Good!

File: students/tao_ye/lesson02/grid_printer.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        print("|" + numUnits*"  " + " |" + numUnits*"  " + " |")
+
+    print("+" + numUnits*" -" + " +" + numUnits*" -" + " +")
+
+    for i in range(numUnits):
+        print("|" + numUnits*"  " + " |" + numUnits*"  " + " |")
+
+    print("+" + numUnits*" -" + " +" + numUnits*" -" + " +")
+
+def print_grid2(numRowCol,numUnits):
```
Comment (charlesrobison) [inline students/tao_ye/lesson02/grid_printer.py:26]: Missing whitespace after comma

File: students/tao_ye/lesson02/grid_printer.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    print("+")
+
+
+if (__name__ == "__main__"):
+    print_grid(3)
+    print_grid(4)
+    print_grid(9)
+
+
+    print_grid2(1,1)
```
Comment (charlesrobison) [inline students/tao_ye/lesson02/grid_printer.py:62]: Missing whitespace after these commas too.

File: students/tao_ye/lesson02/grid_printer.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    print_grid2(3,4)
+    print_grid2(2,5)
```
Comment (charlesrobison) [inline students/tao_ye/lesson02/grid_printer.py:64]: Looks good!

File: students/tao_ye/lesson02/series.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    compute the nth value of a summation series.
+
+    :param n: (int) nth index of the numbers series
+    :Param zero: (int) 0th value of the numbers series
+    :Param one: (int) 1th value of the numbers series
+    :return: (int) nth value of the numbers series
+    
+    """
+    
+    n=int(n)
```
Comment (charlesrobison) [inline students/tao_ye/lesson02/series.py:12]: Missing whitespace around assignment operator

File: students/tao_ye/lesson02/series.py [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+def lucas(n):
+    """
+    compute Lucas numbers series
+
+    :param n: (int) nth index of the Lucas numbers series
+    :return: (int) nth value of the numbers series
+    
+    """
+    
+    return(sum_series(n,2,1))
```
Comment (charlesrobison) [inline students/tao_ye/lesson02/series.py:43]: Need whitespaces after commas here

File: students/tao_ye/lesson03/list_lab.py [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+print("You selected: ", user_choice, "--", fruit[user_choice-1])
+
+fruit = ["Mangos"] + fruit
+print(fruit)
+
+fruit.insert(0, "Grapes")
+print(fruit)
+
+for name in fruit:
+    if (name[0] == "P"):
```
Comment (natasha-aleksandrova) [inline students/tao_ye/lesson03/list_lab.py:22]: no need for parens here -> `if name[0] == "P"`

File: students/tao_ye/lesson03/slicing_lab.py [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    """
+    remove the first 4 and the last 4 items, and keep
+    every other item in the remaining sequence
+
+    :param sequence: a sequence
+    :return: a new sequence
+    
+    """
+    
+    return(sequence[4:-4:2])
```
Comment (natasha-aleksandrova) [inline students/tao_ye/lesson03/slicing_lab.py:34]: no need for parens with return statements

**Pull request merged**